config/peerpods/podvm/bootc/README.md

```diff
@@ -0,0 +1,89 @@
+# Image Mode (bootc) PodVM Builds
+
+Image Mode podVM builds enable OSC users to create PodVM images based
+on a RHEL bootc container image. The resulting artifact can be either
+a podVM bootc container image or a podVM disk generated from that image.
+OSC includes updates to the Image Creation mechanism, supporting the
+conversion and upload of podVM images derived from pre-created bootc
+(image mode) container images.
+The following instructions outline both local and in-cluster options
+for image creation.
+
+## Create PodVM Bootc Container Image
+
+### **Local** PodVM Bootc Container Image Build
+
+Use Podman to build a podVM bootc image locally (push it to
+a container registry of your choice if needed).
+```
+IMG=quay.io/example/podvm-bootc
+AUTHFILE=/path/to/pull-secret
+podman build --authfile ${AUTHFILE} -f Containerfile.rhel -t ${IMG}
+#podman push ${IMG}
+```
+
+### **In-Cluster**  PodVM Bootc Container Image Build
+
+Use an existing OpenShift cluster to execute the container build
+in-cluster and push the container image to the cluster's internal
+registry:
+```
+oc apply -f podvm-git-buildconfig.yaml
+IMG=bootc::image-registry.openshift-image-registry.svc:5000/openshift-sandboxed-containers-operator/podvm-bootc
```

Comment:
IIUC, this IMG variable is not used for the build process. Or it is?

So maybe it would be nice to move it to the "Convert" section?

Reply:
This is pushed to the registry at this point, if it's not clear maybe i can put this in both sections